src/main/java/de/digitalcollections/iiif/hymir/image/business/ImageServiceImpl.java

```diff
@@ -385,6 +388,27 @@ public void processImage(
       String identifier, ImageApiSelector selector, ImageApiProfile profile, OutputStream os)
       throws InvalidParametersException, UnsupportedOperationException, UnsupportedFormatException,
           ResourceNotFoundException, IOException, ScalingException {
+
+    Rectangle2D region = selector.getRegion().getRegion();
+    if (region != null && (region.getWidth() < 1 || region.getHeight() < 1)) {
+      String message =
+          String.format(
+              Locale.ENGLISH,
```

Comment:
ja, ich denke, das ist so ok, da es ja nicht global gesetzt wird, sondern nur für diesen string. Locale.ROOT wäre neutraler, aber keine Ahnung, ob dann Punkte oder Kommas generiert werden (ich vermute Punkte)

Reply:
Is Locale.ROOT now since we want the numbers here formatted independently from the language.